uint8 pin
---
float32 percent
